starters/nuxt-2/pages/products/_handle.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <div v-if="product" class="product">
+    <div v-if="product.content.featuredMedia" class="product__media">
+      <nuxt-img
+        :src="product.content.featuredMedia.src"
+        :alt="product.content.featuredMedia.altText"
+        class="product__image"
+      />
+    </div>
+    <div class="product__main">
+      <h1 v-if="product.content.title" class="product__title">
+        {{ product.content.title }}
+      </h1>
+      <div class="product__prices">
+        <div v-if="selectedVariant.compareAtPrice" class="product__compare">
+          ${{ selectedVariant.compareAtPrice }}
+        </div>
+        <div class="product__price">${{ selectedVariant.price }}</div>
+      </div>
+      <div
+        v-for="(option, oIndex) in options"
+        :key="oIndex"
+        class="product__option"
+      >
+        <label class="product__label">{{ option.name }}</label>
```

Comment:
Same as above:
label needs a :for="id" in order to correctly connect the label to the select